src/styles/components/_global.scss

```diff
@@ -0,0 +1,23 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  color: getColor("main-text");
+}
+
+.container {
```

Comment:
do you really need this block?

Reply:
yes. first: I want to centered block on the page, second: out stiles must be out and in card